multimedia/streamlink/Portfile

```diff
@@ -22,19 +22,23 @@ long_description    ${name} is a CLI utility which pipes video streams from vari
                     ${name} is to avoid resource-heavy and unoptimized websites,\
                     while still allowing the user to enjoy various streamed content.
 
-homepage            https://github.com/streamlink/streamlink/
+homepage            https://streamlink.github.io
 
-python.default_version  38
+python.default_version  312
 
 depends_build-append \
                     port:py${python.version}-setuptools
```

Comment:
this explicit dependency is no longer needed as it will use "python.pep527 yes" with the default "setuptools" backend. Thus the dependency is already added by the "python" PG.

Reply:
Thanks, fixed.